core/src/main/scala/kafka/server/BrokerServer.scala

```diff
@@ -246,9 +245,16 @@ class BrokerServer(
 
       // Create transaction coordinator, but don't start it until we've started replica manager.
```

Comment:
This comment makes more sense if positioned next to the line that initializes the `transactionCoordinator`, right?